assets/css/bootstrap/_map_overrides.scss

```diff
@@ -9,4 +9,5 @@ $theme-colors: (
   "service-alert": semantic.$service-alert,
   "light": new_tokens.$gray-50,
   "dark": new_tokens.$gray-900,
+  "missed-stop": #e8002a,
```

Comment:
suggestion(non-blocking): I hope this isn't too annoying, but maybe we could put this in `tokens_2024` as `missed-stop-1`, and then define `missed-stop` in the `semantic` file via that token?

I think the color belongs in the tokens file because it's become a raw value we're using, but I think it's important that we pull from `semantic` where possible, so it's unfortunate that it'll look like the name is duplicated because the token name and semantic name are the same, but I think it's better in terms of organization in the long run.

Reply:
No, this is exactly the sort of comment I was hoping before because I was personally undecided about where to put it. 😅 I think I'll go with that option unless I suddenly come up with a different counterproposal.

Reply:
Is this what you had in mind: 50565ddf.

Reply:
Love it! Thanks!

All the rest of this comment is non-blocking feedback ATM 👍🏻 

---

I think the only thing I'm not sure about is which "color groups" this should be added to

![image](https://github.com/mbta/skate/assets/6913198/29d4f2b5-4e64-4596-85f3-f30de8dc90bc)
As shown in the above image, it's currently in the `Theme Colors` group, but I'm not really sure it belongs there? But I don't think it belongs in service colors or mbta colors either...

Maaaaaybe it should be in the `Misc Colors > System` group?..
![image](https://github.com/mbta/skate/assets/6913198/81c3c14e-b2fb-4c33-a03e-102ea5e4dc48)

Hard choices 😅

Reply:
Hmm... I agree that in a sense it doesn't make sense as a "theme color" semantically, but it is in fact implemented as a "theme color" in Bootstrap. Maybe this means I shouldn't have done it that way after all? 🙃 But I could probably also do some overrides in `assets/stories/skate-design/colors.ts` to change where the color actually shows up in Storybook.

Reply:
I think it's being added to bootstrap's theme colors, because [I really like how we can use the `bg` parameter due to that choice](https://github.com/mbta/skate/pull/2433#discussion_r1504995254). 

What if we open a ticket to review and potentially fix this with design and merge this PR?

Reply:
I think adding `missed_stop` to the bootstrap theme colors map is a good decision, because [I really like how we can use the `bg` parameter due to that choice](https://github.com/mbta/skate/pull/2433#discussion_r1504995254). But I don't think it necessarily needs to be in that export group though.

I tried playing with it locally and I came up with d1a61c79b5b6978294db15c2c1bef9455851f307, thoughts? (I also realized that my recommendation of `missed-stop-1` was not a great suggestion)

Reply:
Okay, yeah, that seems like a nice balance, I'll go ahead and push it.